types/altv-server.d.ts

```diff
@@ -225,14 +268,41 @@ declare module "alt-server" {
 
     public addWeaponComponent(weaponHash: number, component: number): void;
 
+    /*
+     * Gives a weapon to a player.
+     *
+     * @remarks If a player already has the weaponHash that is to be added, only the ammunation will be increased.
+     * @param weaponHash The weapon hash of the weapon to give.
+     * @param ammo The amount of ammunition to add. Can not be negative.
+     * @param equipNow Automatically select the weapon to give.
+     */
     public giveWeapon(weaponHash: number, ammo: number, equipNow: boolean): void;
 
+    /*
+     * Kicks a player from the server.
+     */
     public kick(): void;
 
+    /*
+     * Removes all weapons from a player.
+     */
     public removeAllWeapons(): void;
 
+    /*
+     * Removes a specific weapon (including ammunition) from a player.
+     *
+     * @remarks If the weapon that should be removed is already removed, this method does nothing.
+     * @param The weapon hash to remove.
```

Comment:
You don't remove a weapon hash, but a weapon that could be get by specified weapon hash